### PR TITLE
CVE 2019-19919 for bootstrap-wysihtml5-rails

### LINKED ISSUE #718: CVE-2019-19919 affects vendored JS library in bootstrap-wysihtml5-rails
The [bootstrap-wysihtml5-rails gem](https://github.com/Nerian/bootstrap-wysihtml5-rails) at v 0.3.3.8 and earlier includes a vendored Javascript library (handlebars v 3.0.2). That library is affected by https://github.com/advisories/GHSA-w457-6q6x-cgp9 and fixed by handlebars v 4.3.0 or 3.0.8. There is no patched version of the bootstrap-wysihtml5-rails gem that updates this version of handlebars.

I've noticed that most of the issues in this database relate to issues in the Ruby code bundled into Ruby gems, but not necessarily issues with vendored Javascript libraries. Is this by design? If so, please close this issue. Otherwise, I can file more issues like this; I've found vulnerable vendored JS in a couple gems.

### PR DESCRIPTION
**Pull request by ddalcino**
Fix #718

Please let me know if you need anything else. Thanks!

### CODE REVIEW

File: gems/bootstrap-wysihtml5-rails/CVE-2019-19919.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+gem: bootstrap-wysihtml5-rails
```
Comment (ddalcino) [inline gems/bootstrap-wysihtml5-rails/CVE-2019-19919.yml:1]:
```suggestion
---
gem: bootstrap-wysihtml5-rails
```

File: gems/bootstrap-wysihtml5-rails/CVE-2019-19919.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ghsa: w457-6q6x-cgp9
+url: https://github.com/advisories/GHSA-w457-6q6x-cgp9
+title: Prototype Pollution in handlebars
+date: 2019-12-26
+description: |
+  The bootstrap-wysihtml5-rails gem includes the vendored JavaScript library 'handlebars.js'.
+  Versions 0.3.3.7-0.3.3.8 include handlebars 3.0.2, and versions 0.3.3.5-0.3.3.6 include handlebars 1.3.0.
+
+  Versions Affected: 0.3.3.5-0.3.3.8
+  Not affected: < 0.3.3.5
```
Comment (postmodern) [inline gems/bootstrap-wysihtml5-rails/CVE-2019-19919.yml:13]: If there are un-affected versions of the gem, than there should be an `unaffected_versions:` attribute with the version range.

**Review by postmodern — CHANGES_REQUESTED** [review]
Probably needs `unaffected_versions:` attribute.

File: gems/bootstrap-wysihtml5-rails/CVE-2019-19919.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  Fixed Versions: None
+
+  Versions of handlebars prior to 3.0.8 or 4.3.0 are vulnerable to Prototype Pollution leading to Remote Code Execution.
+  Templates may alter an Objects' __proto__ and __defineGetter__ properties, which may allow an attacker to execute
+  arbitrary code through crafted payloads.
+unaffected_versions: "< 0.3.3.5"
```
Comment (postmodern) [inline gems/bootstrap-wysihtml5-rails/CVE-2019-19919.yml:19]: Needs to be an Array value.

**Review by postmodern — CHANGES_REQUESTED** [review]
`unaffected_versions:` needs to be an Array.

**Review by postmodern — APPROVED** [review]

**Pull request merged**